UserSync/packages.config

```diff
@@ -20,7 +20,7 @@
   <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net461" />
   <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.3" targetFramework="net461" />
   <package id="Microsoft.AspNet.WebPages" version="3.2.3" targetFramework="net461" />
-  <package id="Microsoft.Identity.Client" version="3.0.3-preview" allowedVersions="[2,3)" targetFramework="net461" />
+  <package id="Microsoft.Identity.Client" version="3.0.8" allowedVersions="[2,4)" targetFramework="net461" />
```

Comment:
This should be [3,4) ? I don't thikn 2 would work here?

Reply:
you are right. fixed.